charts/hivemq-swarm/templates/ingress.yaml

```
@@ -0,0 +1,39 @@
+{{- if .Values.ingress.enabled -}}
+{{- $fullName := include "hivemq-swarm.fullname" . -}}
+{{- $svcPort := .Values.service.port -}}
+{{- if semverCompare ">=1.14-0" .Capabilities.KubeVersion.GitVersion -}}
+apiVersion: networking.k8s.io/v1beta1
+{{- else -}}
+apiVersion: extensions/v1beta1
+{{- end }}
+kind: Ingress
+metadata:
+  name: {{ $fullName }}
```

As above - I propose then to use the namespace functionality in all templates
```suggestion
  name: {{ $fullName }}
  namespace: {{ template "hivemq-swarm.namespace" . }}
```